setup.py

```diff
@@ -20,17 +20,18 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
         'Framework :: Django',
+        'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.0',
+        'Framework :: Django :: 4.1',
+        'Framework :: Django :: 4.2',
         'Framework :: Wagtail',
-        'Framework :: Wagtail :: 1',
-        'Framework :: Wagtail :: 2',
+        'Framework :: Wagtail :: 4.1',
+        'Framework :: Wagtail :: 4.2',
+        'Framework :: Wagtail :: 5.0',
```

Comment:
Wagtail's [PyPI classifiers](https://pypi.org/classifiers/) are only defined against major versions, not minor ones.
```suggestion
        'Framework :: Wagtail :: 4',
        'Framework :: Wagtail :: 5',
```